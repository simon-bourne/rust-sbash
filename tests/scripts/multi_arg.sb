pub fn test-fn(arg1, arg2) {
    echo "$arg1, $arg2"
}
