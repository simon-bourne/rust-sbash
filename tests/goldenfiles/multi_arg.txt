test-fn () { ( arg1="$1"; shift; arg2="$1"; shift; 
    echo "$arg1, $arg2"
) };